dynawo/sources/API/JOB/test/TestXmlImporter.cpp

```diff
@@ -148,7 +148,6 @@ TEST(APIJOBTest, testXmlImporter) {
   ASSERT_EQ(precompiledModelsDirs[0].isRecursive, false);
   ASSERT_EQ(precompiledModelsDirs[1].path, "/tmp/");
   ASSERT_EQ(precompiledModelsDirs[1].isRecursive, true);
-  ASSERT_EQ(preCompiledModelsDirEntry->getModelExtension(), ".so");
```

Comment:
If this entry shouldn't be used anymore, please update the xsd and the API. Otherwise, please make sure this test remains.

Reply:
I will split 
<xs:element name="precompiledModels" type="dyn:ModelsDirEntry"/>
<xs:element name="modelicaModels" type="dyn:ModelsDirEntry"/>
into two xml types because ModelsDirEntry leaves the possibility to have the extension attribute.